hal/BUILD.bazel

```
@@ -0,0 +1,91 @@
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library", "cc_test")
+load("@rules_java//java:defs.bzl", "java_binary", "java_library")
+
+cc_library(
+    name = "generated_cc_headers",
+    hdrs = glob(["src/generated/main/native/include/**"]),
+    includes = ["src/generated/main/native/include"],
+    strip_include_prefix = "src/generated/main/native/include",
+    visibility = ["//hal:__subpackages__"],
+)
+
+filegroup(
+    name = "generated_java",
+    srcs = glob(["src/generated/main/java/**/*.java"]),
+    visibility = ["//hal:__subpackages__"],
+)
+
+ATHENA_SRCS = glob(["src/main/native/athena/**"])
+
+ATHENA_DEPS = ["@bzlmodrio-ni//libraries/cpp/ni:shared"]
+
+SIM_SRCS = glob(["src/main/native/sim/**"])
+
+SIM_DEPS = []
+
+HAL_DEPS = select({
+    "@rules_bzlmodrio_toolchains//constraints/is_roborio:roborio": ATHENA_DEPS,
+    "//conditions:default": SIM_DEPS,
+})
+
+filegroup(
+    name = "platform-srcs",
+    srcs = select({
+        "@rules_bzlmodrio_toolchains//constraints/is_roborio:roborio": ATHENA_SRCS,
+        "//conditions:default": SIM_SRCS,
+    }),
+)
+
+cc_library(
+    name = "wpiHal.static",
+    srcs = ["platform-srcs"] + glob(
```

This should refer to the above `filegroup` target right?
```suggestion
    srcs = [":platform-srcs"] + glob(
```